payroll/models.py

```diff
@@ -241,6 +242,32 @@ def clean(self):
             raise ValidationError('{} name must be unique.'.format(self))
 
 
+class EmployerHighestSalaries(pg.MaterializedView):
```

Comment:
if you declared field mappings in this class, we could interact with it through the ORM. I think that would be desirable.